source/md.h

```diff
@@ -975,7 +975,7 @@ MD_FUNCTION MD_String8     MD_S8(MD_u8 *str, MD_u64 size);
 #endif
 #define MD_S8LitComp(s)     {(MD_u8 *)(s), sizeof(s)-1}
 
-#if MD_CPP_VERSION >= 11
+#if MD_CPP_VERSION != 0 && MD_CPP_VERSION != 98 // anything C++11 and up
```

Comment:
This is changed, because of situation where user invokes compiler with `-std=c++98` or it is default value, like in case for macOS. In such case MD_CPP_VERSION will be set to 98. An the check will perform `98 >= 11` condition which will be true. Which is wrong - you C++98 is not newer than C++11 standard.